tests/Mbstring/MbstringTest.php

```diff
@@ -105,7 +105,7 @@ public function testDecodeNumericEntity()
         $this->assertSame('déjà &#0; â ã', mb_decode_numericentity('déjà &#0; &#225; &#226;', $convmap, 'UTF-8'));
 
         $bogusDecEntities = 'déjà &#0; &#225;&#225; &#&#225&#225 &#225 &#225t';
-        $this->assertSame('déjà &#0; ââ &#&#225â â ât', mb_decode_numericentity($bogusDecEntities, $convmap, 'UTF-8'));
+        $this->assertSame('déjà &#0; ââ &#&#225â â ât', p::mb_decode_numericentity($bogusDecEntities, $convmap, 'UTF-8'));
```

Comment:
This test was affected by changes in the native `mb_decode_numericentity()` function.

https://3v4l.org/fpGFH

It seems that other tests also need to call polyfill methods instead of built-in function.